css/main_styles.css

```diff
@@ -0,0 +1,224 @@
+* {
+  list-style-type: none;
+  font-family: 'Lato', 'Merriweather', sans-serif;
+}
+
+h2 {
+  text-align: center;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+}
+
+#container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+  width: 95%;
+  margin: auto;
+}
+
+nav {
+  grid-column: 1 / 4;
+  padding-bottom: 0;
+  height: 75px;
+  margin: 2rem;
+}
+
+#nav {
+  height: 100%;
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 1rem;
+}
+
+#nav li {
+  width: 20%;
+}
+
+#nav li:first-child {
+  align-self: flex-start;
+  margin-right: auto;
+}
+
+section {
+  border-bottom: 1px solid gray;
+}
+
+#header {
+  grid-column: 1 / 4;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 300px;
+}
+
+#header h1 {
+  font-size: 3rem;
+  margin-bottom: 0.75rem;
+}
+
+#header sub {
+  font-size: 1.5rem;
+  margin-top: 0;
+  padding-top: 0;
+  text-transform: capitalize;
+  font-weight: bold;
+}
+
+#header a {
+  width: auto;
+  margin: 1rem auto 2rem auto;
```

Comment:
Great job assigning a small rounded border to the "Some Call to Action" and having it fit perfectly around those words.